util/getUserBitnessByArchitecture.ts

```diff
@@ -0,0 +1,10 @@
+export const getUserBitnessByArchitecture = (
+  userArchitecture: string,
+  userBitness: number
+) => {
+  if (userArchitecture == 'arm' && userBitness == 64) {
```

Comment:
nit: should these be triple equals checks? especially with this being a function that is checking strings and number input. i suppose TypeScript gives us some help here but it stood out.
```suggestion
  if (userArchitecture === 'arm' && userBitness === 64) {
```